uamqp/receiver.py

```diff
@@ -74,9 +74,13 @@ def _incoming_transfer(self, frame):
         if not self.received_delivery_id and not self._received_payload:
             pass  # TODO: delivery error
         if self._received_payload or frame[5]:  # more
-            raise NotImplementedError()  # TODO
+            self._received_payload += bytes(frame[11])  # TODO, is it possible to use memoryview directly for concatenation?
```

Comment:
It occurs to me that we need to be clear in our documentation/SDK implementations that a Receiver is not threadsafe....

Reply:
Yeah - I think bytes/memoryview issue might be why I punted this....
You could start with what you have here, to get it functional, leave the "TODOs" in place and then we can revisit this during the perf testing process.

Reply:
I just put a comment on the client classes docstring mentioning not thread safe, also add an onenote page about the docs.

for receive payload, depending on the send perf PR, I think same bytearray implementation could be applied here, but that would be in a separate PR.